src/activateStart.ts

```diff
@@ -0,0 +1,15 @@
+// import { config } from '../styles/SASS/config.scss'
```

Comment:
I tried to import variables from config.scss. For this purpose I appiled following schame but it does not work
[https://til.hashrocket.com/posts/sxbrscjuqu-share-scss-variables-with-javascript](url)

@lukaszdutka Do you know how to use it with TS?

Reply:
It looks like it needs React. I don't know if it is possible to share variables to javascript, but:
workaround (or even better approach) is to apply different style class (i.e. for button: `button` and `button-disabled`) and changing classes in code is more appropriate :)

Reply:
See here: https://clubmate.fi/javascript-adding-and-removing-class-names-from-elements

Reply:
Done :)